src/screens/Sidebar/LinkedAccounts.js

```diff
@@ -51,45 +51,21 @@ class LinkedAccounts extends React.Component {
   render() {
     const { navigation } = this.props;
     return (
-      // handle marginTop: 77
-      (
-        <Container style={styles.containerStyle}>
-          <Content scrollEnabled={false}>
-            <View style={{ flex: 1, marginTop: 77 }}>
-              <SidebarTitle title={'Social Accounts'} />
-              <FlatList
-                data={[{ logoURL: 'https://www.famouslogos.us/images/facebook-logo.jpg' }]}
-                keyExtractor={(item, index) => index}
-                renderItem={this.renderItem}
-                ItemSeparatorComponent={() => <ItemSeparator />}
-                removeClippedSubviews={false}
-                scrollEnabled={false}
-              />
-            </View>
-          </Content>
-        </Container>
-      )
+      <View style={styles.containerStyle}>
+        <SidebarTitle title={'Social Accounts'} />
+        <FlatList
+          data={[{ logoURL: 'https://www.famouslogos.us/images/facebook-logo.jpg' }]}
+          keyExtractor={(item, index) => index}
+          renderItem={this.renderItem}
+          ItemSeparatorComponent={() => <ItemSeparator />}
+          removeClippedSubviews={false}
+          scrollEnabled={false}
+        />
+      </View>
```

Comment:
@edencakir you can use ScrollView instead of FlatList –– If you're certain about the number of child components.

Reply:
@BilalBudhani I'll rewrite this file later, just a dummy UI.